test/metabase/pulse/render/js_svg_test.clj

```diff
@@ -112,21 +112,21 @@
   (let [rows        [[#t "2020" 2]
                      [#t "2021" 3]]
         series-seqs [[{:type          :line
-                       :color         "#999AC4"
                        :data          rows
                        :yAxisPosition "left"
                        :column        {:name "count"}}]]
-        settings    {:colors {:brand     "#5E81AC"
-                              :filter    "#A3BE8C"
-                              :summarize "#B48EAD"},
-                     :x      {:type   "timeseries"
-                              :format {:date_style "YYYY/MM/DD"}}
-                     :y      {:type   "linear"
-                              :format {:prefix   "prefix"
-                                       :decimals 2}}
-                     :labels {:bottom ""
-                              :left   ""
-                              :right  ""}}
+        settings    {:colors                 {:brand     "#5E81AC"
+                                              :filter    "#A3BE8C"
+                                              :summarize "#B48EAD"},
+                     :x                      {:type   "timeseries"
+                                              :format {:date_style "YYYY/MM/DD"}}
+                     :y                      {:type   "linear"
+                                              :format {:prefix   "prefix"
+                                                       :decimals 2}}
+                     :labels                 {:bottom ""
+                                              :left   ""
+                                              :right  ""}
+                     :visualization_settings {}}
```

Comment:
Now that I always pass `:visualization_settings` to FE and it expects this value to always be present, I need to fix all the tests here.

I could make FE check if this value is null, but in reality, it will always have a value, so making that change in favor of not having to fix tests, seems a bit misleading.